packages/cadl-ranch-specs/http/payload/media-type/main.tsp

```diff
@@ -0,0 +1,55 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+/**
+ * Test the payload with different media types and different types of the payload itself.
+ */
+@supportedBy("dpg")
+@scenarioService("/payload/media-type")
+namespace Payload.MediaType;
+
+@scenario
+@scenarioDoc("""
+  When the request body and response body are both strings, the different media types would lead to different payloads.
```

Comment:
What do you mean different media types lead to different payloads? It seems all the payload is just string, the difference is only the content type.

Reply:
You could see details from this customer request https://github.com/microsoft/typespec/discussions/2557

You could think the media type as how we encode/decode the content. For example, when the media type is "text/plain", we just keep what the data is, not do anything like adding quotes or deleting quotes as what "application/json" does.

Reply:
I see. Seems content negotiation is for response, this is for request.

Reply:
Yes, customer request is for response, but here in tests I want to make sure we test request as well.